designs/index.md

```diff
@@ -221,29 +232,6 @@ Blaze manager for further disposition.
     its current state, and outlining next steps, if any (e.g., "revisit invalid
     assumptions and resubmit").
 
-
-## What is a significant change?
-
-
-There are no hard and fast criteria, but here are some examples:
-
-*   Addition or deletion of native build rules
-*   Breaking-changes to native rules
-*   Changes to a native build rule semantics that affect the behavior of more
-    than a single rule
-*   Changes to Bazel's rule definition API
-*   Changes to the APIs that Bazel uses to connect to other systems
-*   Changes to the Starlark language, semantics, or APIs
-*   Changes that could have a pervasive effect on Bazel performance or memory
-    usage (for better or for worse)
-*   Changes to widely used internal APIs
-*   Changes to flags and command-line interface.
-
-When a proposal adds, removes, or modifies any function or object available in
-BUILD, WORKSPACE, or bzl files, the [Starlark team](../maintainers-guide.html)
-has to be in the reviewers list.
-
-
 ## Why do we need design reviews?
```

Comment:
These might be duplicative of the introductory text that covers why contributors should make design docs. Consider folding them in.

Reply:
Moved this section up